### ISSUE TITLE
Arxiv Paper Digest for 2025-01-17

### ISSUE THREAD

**Issue by mismayil** [mismayil/daily-arxiv-digest#41]
### Convex Markov Games: A Framework for Creativity, Imitation, Fairness, and Safety in Multiagent Learning
 - **Authors:** Ian Gemp, Andreas Haupt, Luke Marris, Siqi Liu, Georgios Piliouras
 - **Subjects:** Computer Science and Game Theory (cs.GT), Artificial Intelligence (cs.AI), Multiagent Systems (cs.MA)
 - **Arxiv link:** https://arxiv.org/abs/2410.16600
 - **Pdf link:** https://arxiv.org/pdf/2410.16600
 - **Abstract**
 Behavioral diversity, expert imitation, fairness, safety goals and others give rise to preferences in sequential decision making domains that do not decompose additively across time. We introduce the class of convex Markov games that allow general convex preferences over occupancy measures. Despite infinite time horizon and strictly higher generality than Markov games, pure strategy Nash equilibria exist. Furthermore, equilibria can be approximated empirically by performing gradient descent on an upper bound of exploitability. Our experiments reveal novel solutions to classic repeated normal-form games, find fair solutions in a repeated asymmetric coordination game, and prioritize safe long-term behavior in a robot warehouse environment. In the prisoner's dilemma, our algorithm leverages transient imitation to find a policy profile that deviates from observed human play only slightly, yet achieves higher per-player utility while also being three orders of magnitude less exploitable.

### TextureCrop: Enhancing Synthetic Image Detection through Texture-based Cropping
 - **Authors:** Despina Konstantinidou, Christos Koutlis, Symeon Papadopoulos
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2407.15500
 - **Pdf link:** https://arxiv.org/pdf/2407.15500
 - **Abstract**
 Generative AI technologies produce increasingly realistic imagery, which, despite its potential for creative applications, can also be misused to produce misleading and harmful content. This renders Synthetic Image Detection (SID) methods essential for identifying AI-generated content online. State-of-the-art SID methods typically resize or center-crop input images due to architectural or computational constraints, which hampers the detection of artifacts that appear in high-resolution images. To address this limitation, we propose TextureCrop, an image pre-processing component that can be plugged in any pre-trained SID model to improve its performance. By focusing on high-frequency image parts where generative artifacts are prevalent, TextureCrop enhances SID performance with manageable memory requirements. Experimental results demonstrate a consistent improvement in AUC across various detectors by 6.1% compared to center cropping and by 15% compared to resizing, across high-resolution images from the Forensynths, Synthbuster and TWIGMA datasets. Code available at https : //github.com/mever-team/texture-crop.
